src/SignalR/clients/ts/signalr/src/Utils.ts

```diff
@@ -181,3 +187,72 @@ export class ConsoleLogger implements ILogger {
         }
     }
 }
+
+/** @private */
+export function getUserAgentHeader(): [string, string] {
+    let userAgentHeaderName = "X-SignalR-UserAgent";
```

Comment:
Why X-SignalR-UserAgent instead of X-SignalR-User-Agent to be more consistent with the original value?

Reply:
I like that better :+1: